README.rst

```diff
@@ -30,18 +30,24 @@ Log out and in again and clone this repository::
   $ git clone https://github.com/rm-hull/luma.examples.git
   $ cd luma.examples
 
-Finally, install the luma libraries using::
+Next, create a `virtual environment <https://docs.python.org/3/library/venv.html>`__::
 
-  $ sudo -H pip install -e .
+  $ python3 -m venv ~/luma-env
```

Comment:
I tend to use:

```
python3 -m venv .venv
.venv/bin/activate
```

I think IDEs like vscode and tools like pyenv will sniff out a `.venv` directory and will auto-activate it.

Is there any reason not to stick to that convention?

Reply:
Or are you aiming to pull all the dependencies for all sub-projects into one place?

Reply:
> Is there any reason not to stick to that convention?

I didn't know that `IDEs like vscode and tools like pyenv will sniff out a .venv directory and will auto-activate it`, but maybe I'm just old-skool :-/

Reply:
> Is there any reason not to stick to that convention?

I like to be know where my virtual envs live, explicitly, in this case `~/luma-env`, and since I didn't know about that convention (and new users wont either i'd guess) I chose this.